warehouse/integrations/clickhouse/clickhouse.go

```diff
@@ -491,6 +503,9 @@ func (ch *Clickhouse) typecastDataFromType(data, dataType string) interface{} {
 
 // loadTable loads table to clickhouse from the load files
 func (ch *Clickhouse) loadTable(ctx context.Context, tableName string, tableSchemaInUpload model.TableSchema) (err error) {
+	if err = misc.SleepCtx(ctx, ch.config.randomLoadDelay(ch.Warehouse.Destination.ID)); err != nil {
+		return
+	}
```

Comment:
[minor]: Should we do delay only in case some delay is present?
```suggestion
	if delay := ch.config.randomLoadDelay(ch.Warehouse.Destination.ID); delay > 0 {
		if err = misc.SleepCtx(ctx, delay); err != nil {
			return
		}
	}
```

Reply:
wouldn't `time.After` take care of it anyway..?
Meaning if the delay is zero, there's no delay😅